docs/builds/guides/integration/advanced-setup.md

```diff
@@ -143,16 +143,11 @@ module.exports = {
 	module: {
 		rules: [
 			{
-				// Or /ckeditor5-[^/]+\/theme\/icons\/[^/]+\.svg$/ if you want to limit this loader
-				// to CKEditor 5 icons only.
-				test: /\.svg$/,
-
+				test: /ckeditor5-[^/\\]+[/\\]theme[/\\]icons[/\\][^/\\]+\.svg$/,
```

Comment:
It deserves a comment. Something like:

> Process CKEditor 5 icons (ckeditor5-*/theme/icons/*.svg) using raw-loader.

This comment makes sense before `{`, just like you did in the below example in the new section.

I'm also thinking about simplifying this regexp to:

```
/ckeditor5-[^/\\]+[/\\]theme[/\\]icons[/\\].+\.svg$/
```

It will cover a bit more cases (e.g. `theme/icons/some-group-of-tem/foo.svg`).